frameworks/digital-outcomes-and-specialists-2/manifests/edit_service_as_admin.yml

```diff
@@ -1,5 +1,124 @@
--
-  name: Description
-  editable: true
+- name: Lab name
```

Comment:
~~Might be worth grouping this with all the "Lab" questions towards the bottom of the manifest (only labs have `serviceName` defined).~~

Actually, this just follows the order in `edit_submission.yml` so let's stick with it as it is.